lib/services/game_service.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:niira/models/game.dart';
+
+class GameService {
+  Game currentGame;
+  GameService({this.currentGame});
+
+  set setCurrentGame(Game _selectedGame) => currentGame = _selectedGame;
```

Comment:
Yo! 

So using the underscore in the parameter (ie. _selectedGame) doesn't do anything as far as I can tell - it doesn't live on outside the setter so there's no reason to make it "library private" as far as I can tell - am I missing something? 

Do you want the `currentGame` member to be "library private"?  If yes, you'll need getters and setters, if no you can just use the default getter and setter. 

`set setCurrentGame(...` seems like clunky syntax to me - it would mean we would be doing something like `service.setCurrentGame = _newGame` which I think is non-standard and a bit confusing, what do you think?

Reply:
above comments are for outdated code

Reply:
alg thanks nick!